tooling/security-headers.js

```
@@ -0,0 +1,64 @@
+    { key: 'Strict-Transport-Security', value: 'max-age=63072000; includeSubDomains; preload' },
+    { key: 'X-Frame-Options', value: 'DENY' },
+    { key: 'X-Content-Type-Options', value: 'nosniff' },
+    { key: 'Referrer-Policy', value: 'strict-origin-when-cross-origin' },
+    { key: 'X-DNS-Prefetch-Control', value: 'off' },
+    // Nothing in these apps uses the camera or mic. Geolocation stays available:
+    // the provider app tracks live location.
+    { key: 'Permissions-Policy', value: 'camera=(), microphone=(), payment=(self "https://js.stripe.com")' },
+  ];
+}
+
+module.exports = { securityHeaders, contentSecurityPolicy };
```

ESLint

[error] 'module' is not defined.

(no-undef)